validator/src/validator.rs

```diff
@@ -211,6 +213,8 @@ impl<TNetwork: Network, TValidatorNetwork: ValidatorNetwork>
             mempool_state,
             #[cfg(feature = "metrics")]
             mempool_monitor: TaskMonitor::new(),
+            #[cfg(feature = "metrics")]
+            ctrl_mempool_monitor: TaskMonitor::new(),
```

Comment:
Suggestion to rename, to be consistent across the codebase.
```suggestion
            control_mempool_monitor: TaskMonitor::new(),
```